src/code-view/styles.scss

```diff
@@ -5,62 +5,72 @@ $color-background-code-view-dark: #282c34;
 
 .root {
   position: relative;
-  &-with-numbers {
-    display: flex;
-    align-items: stretch;
-  }
-}
-
-.code {
   :global(.awsui-dark-mode) &,
   :global(.awsui-polaris-dark-mode) & {
     background-color: $color-background-code-view-dark;
   }
-  &-with-line-numbers {
-    border-start-start-radius: 0;
-    border-end-start-radius: 0;
-    flex: 1;
-  }
-  &-with-actions {
-    min-block-size: cs.$space-scaled-xxl;
-    padding-inline-end: calc(2 * cs.$space-static-xxxl);
-    align-items: center;
-  }
   background-color: $color-background-code-view-light;
-  display: flex;
+}
+
+.scroll-container {
+  overflow-x: auto;
+}
+
+.code-table {
   border-start-start-radius: cs.$border-radius-tiles;
   border-start-end-radius: cs.$border-radius-tiles;
   border-end-start-radius: cs.$border-radius-tiles;
   border-end-end-radius: cs.$border-radius-tiles;
-  padding-block: cs.$space-static-xs;
-  padding-inline: cs.$space-static-xs;
-  margin-block: 0;
-  margin-inline: 0;
-  overflow: auto;
+  padding-top: cs.$space-static-xs;
```

Comment:
should be RTL-compliant (`padding-block-start` etc.)

Reply:
Updated this.